### PR TITLE
Require Faker in seeds file

### PR DESCRIPTION
**Pull request by Eduardo06sp**
### Describe your change in plain English.

I was trying to reset the database (`bin/rails db:reset`), but I was getting an error due to Faker being used in the seeds file. It is not `require`d anywhere in the seeds file, so this fixes that.

### CODE REVIEW

**Comment by edwinthinks** [comment]
Woopsie we forgot to put the Gem record in both test and development. That should address this.

**Comment by Thrillberg** [comment]
Hopefully this is [resolved by this PR](https://github.com/rubyforgood/pet-rescue/pull/47). @Eduardo06sp can you let us know if it still breaks after that gets merged?

**Comment by Eduardo06sp** [comment]
@Thrillberg That indeed resolves the issue! Thanks for catching that PR!
